apps/emqx_exproto/src/emqx_exproto_channel.erl

```diff
@@ -159,7 +159,7 @@ init(ConnInfo = #{socktype := Socktype,
     Channel = #channel{gcli = #{channel => GRpcChann},
                        conninfo = NConnInfo1,
                        clientinfo = ClientInfo,
-                       conn_state = accepted,
+                       conn_state = idle,
```

Comment:
should fix the spec instead.

Reply:
there is a at least one special handling for the `accepted` state in `emqx_mgmt_api_clients.erl`, which treats `accepted` as `connected`.
Not sure if it's used in any other places.